tests/serverpod_test_server/test_integration/database_operations/default/integer/int_default_mix_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_server/src/generated/protocol.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+void main() async {
+  var session = await IntegrationTestServer().session();
+
+  group('Given a class with mixed default fields,', () {
+    tearDownAll(() async => IntDefaultMix.db.deleteWhere(
+          session,
+          where: (_) => Constant.bool(true),
+        ));
+
+    test(
+      'when creating a record in the database with an unsafe query, then the "intDefaultAndDefaultModel" field value should match the provided value',
+      () async {
```

Comment:
Suggestion: Same comment here as for the bool test. The description does not match the test.